src/main/java/org/tikv/common/PDClient.java

```diff
@@ -356,9 +356,16 @@ private boolean isScatterRegionFinish(GetOperatorResponse resp) {
 
   @Override
   public Store getStore(BackOffer backOffer, long storeId) {
-    return callWithRetry(
-            backOffer, PDGrpc.getGetStoreMethod(), buildGetStoreReq(storeId), buildPDErrorHandler())
-        .getStore();
+    GetStoreResponse resp =
+        callWithRetry(
+            backOffer,
+            PDGrpc.getGetStoreMethod(),
+            buildGetStoreReq(storeId),
+            buildPDErrorHandler());
+    if (resp != null) {
+      return resp.getStore();
+    }
+    return null;
```

Comment:
could you add some comments about:

1. the possible scenarios where get store fails with a null response
2. why not throw an exception in those scenarios